spec/presenters/two_factor_auth_code/phone_delivery_presenter_spec.rb

```diff
@@ -125,4 +144,12 @@
       end
     end
   end
+
+  def presenter_with_locale(locale)
+    TwoFactorAuthCode::PhoneDeliveryPresenter.new(
+      data: data.clone.merge(reenter_phone_number_path:
```

Comment:
Why call `#clone` here? `Hash#merge` should return a new hash without modifying the underlying hash.

Reply:
good catch.  originally I had the clone on a separate line from the merge because I thought I was going to do more with the hash